### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by nickanderson**
- Add skeleton for acceptance test in docs
- Cleanup whitespace (kill tabs) in reports example

### CODE REVIEW

File: examples/reports.cf [hunk 1]
```diff
@@ -3,7 +3,7 @@ bundle agent main
 {
   reports:
       "It's reccomended that you always guard reports"
-        comment => "Remember by default output from cf-agent when run
+	comment => "Remember by default output from cf-agent when run
 		    from cf-execd will be emailed";
```
Comment (jimis) [inline examples/reports.cf]: Why change the two spaces to one?

File: tests/acceptance/README [hunk 2]
```diff
@@ -258,3 +258,41 @@ Soft fail: the test failed as expected by a "test_soft_fail" promise.
...
+
+}
+
+bundle agent test
+# Activate policy for behaviour you wish to inspect
+{
+  meta:
+    "description" string => "What does this test?";
+    "story_id" string => "5525";
+    "covers" string => "operational_repaired";
```
Comment (jimis) [inline tests/acceptance/README]: "story_id" and "covers" is a bit confusing and useless for community contributions. Maybe just remove?

File: tests/acceptance/README [hunk 3]
```diff
@@ -258,3 +258,41 @@ Soft fail: the test failed as expected by a "test_soft_fail" promise.
...
+
+    "test_soft_fail"
```
Comment (jimis) [inline tests/acceptance/README]: come comment on what this does, also another entry on how to skip a platform would also be useful.

File: examples/reports.cf (hunk 1 above)
Comment (jimis) [inline examples/reports.cf]: Err, you changed spaces to tabs. Why?

**Comment by jimis** [comment]
@nickanderson where are you?

**Comment by nickanderson** [comment]
I am at home :palm_tree:

Will look at this.

File: examples/reports.cf (hunk 1 above)
Comment (nickanderson) [inline examples/reports.cf]: Because editor woes, will fix.

**Comment by nickanderson** [comment]
ping @jimis WHERE ARE YOU?

**Comment by jimis** [comment]
Un-assigning from me because of lack of time. Sorry, but any other reviewer feel free to review ...

**Comment by kacf** [comment]
I don't think the example changes can influence the test, so I'll risk merging this directly. Oh, the thrill!

**Pull request merged**